### PR TITLE
Add new License type enum value for DR

### PR DESCRIPTION
**Pull request by ranishan**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2017-03-01-preview/sqlvm.json [hunk 1]
```diff
@@ -1397,7 +1397,8 @@
           "description": "SQL Server license type.",
           "enum": [
             "PAYG",
-            "AHUB"
+            "AHUB",
+            "DR"
```
Comment (raych1) [inline specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2017-03-01-preview/sqlvm.json]: Adding new enum value is breaking change, prefer to change in new version.

**Review by raych1 — CHANGES_REQUESTED** [review]
@ranishan , please resolve the model validation [failures](https://dev.azure.com/azure-sdk/public/_build/results?buildId=193539&view=logs&jobId=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&j=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&t=43324f1f-6724-5d61-785f-f0ac9fd256a1).

File: specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2017-03-01-preview/sqlvm.json (hunk 1 above)
Comment (ranishan) [reply specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2017-03-01-preview/sqlvm.json]: I will not classify it as a breaking change as we are not adding a new property as such. 
We are adding a new value to an existing Enum and we are not changing the older values. So this should be a backward compatible change.
We would want to keep change in the current version. We are not planning in any new version releases soon and we cannot hold on to this change due to the project timeline.

**Comment by ranishan** [comment]
Resolved the failures. @raych1 Please review.

**Review by raych1 — APPROVED** [review]

**Pull request merged**